moveit_commander/src/moveit_commander/planning_scene_interface.py

```diff
@@ -151,7 +151,7 @@ def add_plane(self, name, pose, normal = (0, 0, 1), offset = 0):
         
     def attach_mesh(self, link, name, pose = None, filename = '', size = (1, 1, 1), touch_links = []):
         aco = AttachedCollisionObject()
-        if pose!=None and not filename.empty():
+        if pose!=None and not filename:
```

Comment:
I don't know python enough to understand why this fix works, seems counter intuitive